flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -591,5 +729,27 @@ object AggregateUtil {
 
     groupingOffsetMapping.toArray
   }
+
+  private def getTimeFieldPosition(timeField: Expression, inputType: RelDataType): Int = {
+    timeField match {
+      case ResolvedFieldReference(name, resultType) =>
+        // get the RelDataType referenced by the time-field
+        val relDataType = inputType.getFieldList.filter(r => name.equals(r.getName))
```

Comment:
We set `Lex.JAVA` as default lexical policy which is case-sensitive. I will change this logical to adapt case-sensitive or insensitive. So that it works when user customize lexical policy.